Doc/library/importlib.metadata.rst

```diff
@@ -178,7 +178,7 @@ The "selectable" entry points were introduced in ``importlib_metadata``
 no parameters and always returned a dictionary of entry points, keyed
 by group. With ``importlib_metadata`` 5.0 and Python 3.12,
 ``entry_points`` always returns an ``EntryPoints`` object. See
-`backports.entry_points_selectable <https://pypi.org/project/backports.entry_points_selectable>`_
+`backports.entry-points-selectable <https://pypi.org/project/backports.entry-points-selectable>`_
```

Comment:
```suggestion
`backports.entry_points_selectable <https://pypi.org/project/backports.entry-points-selectable>`_
```

The PyPI name is normalised to dashes, but the project keeps underscores in its own text.

* https://pypi.org/project/backports.entry-points-selectable/
* https://github.com/jaraco/backports.entry_points_selectable
* https://github.com/jaraco/backports.entry_points_selectable/blob/0d7e54c153a5fee01b9c69a03509e814225954f1/setup.cfg#L2